test/verify/check-connection

```diff
@@ -459,32 +459,57 @@ class TestConnection(MachineCase):
     def testSocket(self):
         m = self.machine
 
-        self.assertIn("systemctl", m.execute("cat /etc/issue.d/cockpit.issue"))
-        self.assertIn("systemctl", m.execute("cat /etc/motd.d/cockpit"))
-        self.assertNotIn("9090", m.execute("cat /etc/motd.d/cockpit"))
-        m.start_cockpit()
+        # non-admin user
+        m.execute("useradd user")
+
+        # enable no-password login for 'admin' and 'user'
+        m.execute("passwd -d admin")
+        m.execute("passwd -d user")
+        self.sed_file('$ a\\\nPermitEmptyPasswords yes', '/etc/ssh/sshd_config',
+                      'systemctl restart sshd.service')
+
+        def assertInOrNot(string, result, expected):
+            if expected:
+                self.assertIn(string, result)
+            else:
+                self.assertNotIn(string, result)
+
+        def checkMotdForUser(string, user, expected):
+            result = m.execute(f"ssh -o StrictHostKeyChecking=no -n {user}@localhost")
+            assertInOrNot(string, result, expected)
+
+        def checkMotdContent(string, expected=True):
+            pam_checks_perms = (m.image in ['fedora-33'])
 
-        self.assertNotIn("systemctl", m.execute("cat /etc/motd.d/cockpit"))
-        self.assertIn("9090", m.execute("cat /etc/issue.d/cockpit.issue"))
-        self.assertIn("9090", m.execute("cat /etc/motd.d/cockpit"))
+            # check issue (should be exactly the same as motd)
+            assertInOrNot(string, m.execute("cat /etc/issue.d/cockpit.issue"), expected)
+
+            # check motd as 'root' (via cat) and 'admin' and 'user' (via ssh)
+            assertInOrNot(string, m.execute("cat /etc/motd.d/cockpit"), expected)
+            checkMotdForUser(string, expected=expected, user='admin')
```

Comment:
I am a bit confused why this works on F-33? I thought https://bugzilla.redhat.com/show_bug.cgi?id=1896452 was about exactly this case, that an admin user does not see motd *even though* it is in the `wheel` group. So doesn't that special case need to be handled here? And the more general "OS has recent enough PAM for privilege dropping" needs to be a separate list, and handled in the `user='user'` case?

Reply:
The [test failure on f-33](https://logs.cockpit-project.org/logs/pull-14856-20201105-114809-f4f0a4bb-fedora-33/log.html#308) confirms that this is indeed broken.

Reply:
The tests are broken on fedora-33, intentionally.  I'm happy to leave them that way until we have a new fedora-33 image, at which point they should magically work.

Reply:
Of course, sorry. So the blocked label and todo list were right, and I got confused about the release-blocker label. Thanks!